### ISSUE TITLE
Sweeper "minigun" exploit

### ISSUE THREAD

**Issue by DracTheGreat** [JetBoom/zombiesurvival#135]
You can spam the reload button after shooting with the sweeper to completely remove the fire delay. Pretty game breaking imo

**Comment by Zverina** [comment]
Well, seeing as how it is innacurate and does pretty normal shotgun damage per shell, this seems fine to me for a 200 point weapon.

**Comment by Zverina** [comment]
Misread. Completely remove fire delay? I have not noticed this. 

I thought you meant kind of like Beggar's Bazooka from TF2.

**Comment by DracTheGreat** [comment]
It's not innacurrate at all. It completely destroys zombies at close range. This IS a completely game break exploit, and you haven't noticed it because very few people know about it.

**Comment by JetBoom** [comment]
Are you sure this actually happens? I cannot replicate it.

**Comment by DracTheGreat** [comment]
I'm sure it does. Maybe this'll help?

```
[10:53:20 PM] dude: you load less than the max shells you can load
[10:53:25 PM] dude: this works on blaster and annabelle too
[10:53:26 PM] dude: btw
[10:53:36 PM] dude: for annabelle its 3, blaster is 4, sweeper is 5
[10:53:46 PM] dude: thats the maximum you must load for the exploit to work
[10:54:02 PM] dude: did the server just crash?
[10:54:11 PM] dude: ok
[10:54:21 PM] dude: right after you fire
[10:54:23 PM] dude: you reload
[10:54:28 PM] dude: fire once again to pump
[10:54:36 PM] dude: then fire once more
[10:54:42 PM] dude: the delay between shots should be much lower
[10:55:16 PM] dude: works on all zs servers
[10:55:38 PM] dude: and for some reason if you do it with 0 shells loaded the exploit doesnt work as well
```

It also makes it so you never have to reload.

**Comment by Intificial** [comment]
It's also possible to do this with the tiny slug. You can effectively "fit the reload animation" into the fire delay, allowing you to "never reload". Hit reload as soon as you fire, it's fairly easy to replicate.

**Comment by Seba0702** [comment]
This happens to my server too. i had to remove the sweeper

**Comment by C0nw0nk** [comment]
I did a fix here what allot of people use i am not sure what jetboom's fix in the next update is but nobody has had any issues with mine.

https://github.com/C0nw0nk/zombiesurvival/blob/master/gamemodes/zombiesurvival/entities/weapons/weapon_zs_sweepershotgun.lua#L104

**Comment by JetBoom** [comment]
Fixed in feb2016 update.
